index.html

```diff
@@ -4,35 +4,80 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./style.css"/>
     <title>YouTube Embedded Video Page</title>
   </head>
   <body>
-    <header class="home-search">
-      <img src="index.jpg" alt="YouTube image" />
-      <input
-        type="text"
-        name="search bar"
-        title="search bar"
-        placeholder="Search"
-      />
-      <button type="submit"><i class="fas fa-search"></i></button>
+    <header id="nav-bar">
+      <img id="logo" src="index.jpg" alt="YouTube image" />
+      <div id="searchView">
+        <input
+          type="text"
+          name="search bar"
+          title="search bar"
+          placeholder="Search"
+        />
+        <button type="submit"><i class="fas fa-search"></i></button>
+      </div>
+      <div id="rightNavView">
+        <button id="searchButton" type="submit"><i class="fas fa-bars"></i></button>
+        <button id="searchButton" type="submit"><i class="fas fa-ellipsis-v"></i></button>
+        <button id="searchButton" type="submit"><i class="fas fa-upload"></i></button>
```

Comment:
The id attribute can not be identical on two elements, use the class attribute instead